api/block/schema.py

```diff
@@ -0,0 +1,28 @@
+import graphene
+from graphene import relay,Schema
+from graphene_sqlalchemy import (
+    SQLAlchemyObjectType, 
+    SQLAlchemyConnectionField
+)
+from api.block.models import Block as BlockModel
+
+class Block(SQLAlchemyObjectType):
+    class Meta:
+        model = BlockModel
+
+class Query(graphene.ObjectType):
+    node = relay.Node.Field()
+    floor = graphene.List(Block)
+    get_rooms_in_a_block = graphene.List(
```

Comment:
I think this should be **get_block** not **get_rooms_in_a_block**

Reply:
Thanks. The flow is being followed as it necessitates by the front-end flow.